Bottom: 8d05a35b98a0de6cfac35e36537c48c64a75af3f
Top:    61815c9c6be5c0fd87efa33cd65cf62d617d31ce
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-24 14:31:12 -0700

rsockets: Change the default QP size from 512 to 384

Simple bandwidth tests using rstream showed no difference in
performance between using a QP sized to 384 entries versus 512.
Reduce the overhead of a default rsocket by using 384 entries.
A user can request a larger size by calling rsetsockopt.

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/src/rsocket.c b/src/rsocket.c
index 7802ef5..8a3fc60 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -54,7 +54,7 @@
 #define RS_INLINE 64
 #define RS_OLAP_START_SIZE 2048
 #define RS_MAX_TRANSFER 65536
-#define RS_QP_SIZE 512
+#define RS_QP_SIZE 384
 #define RS_QP_MAX_SIZE 0xFFFE
 #define RS_QP_MIN_SIZE 8
 #define RS_QP_CTRL_SIZE 4
